test/unit/assorted/uri_options.spec.test.ts

```diff
@@ -0,0 +1,61 @@
+import { loadSpecTests } from '../../spec';
+import { executeUriValidationTest } from '../../tools/uri_spec_runner';
+
+describe('URI option spec tests', function () {
+  const suites = loadSpecTests('uri-options');
+
+  const skipTests = [
+    // TODO: fix?
```

Comment:
both of these are supposed to be invalid but aren't, TODO: fix

Reply:
Ah I see the issue there with the loadBalanced + directConnection combo: https://github.com/mongodb/node-mongodb-native/blob/main/src/connection_string.ts#L495 - it will only error when directConnection is truthy.

Reply:
Anna beat me to filing NODE-3917 for the direct connection issue, I think we can tackle all these things in that one ticket